pilot/pkg/bootstrap/namespacecontroller.go

```diff
@@ -74,6 +75,11 @@ func NewNamespaceController(data func() map[string]string, kubeClient kubernetes
 			options.LabelSelector = fields.SelectorFromSet(configMapLabel).String()
 		})
 	configmapInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
```

Comment:
Why do we need to look at an add? Is this for a case where someone creates an empty configmap with the istio-root-ca name, or something more?

Reply:
I wasn't thinking of a specific case. I could imagine some potentially contrived cases, e.g. multiple istiod instances race to create a new configmap. One of the istiod is from an other replicaset which has a bug writing the configmap. If that buggy istiod wins the race, it would never by the other new/fixed istiod instances. 

Instead of thinking of specific cases, it was easier guaranteed the configmap would always eventually be correct. If the configmap is added for whatever reason, it must have the correct istio-root-ca.